### PR TITLE
Paragraph collapsible panel item cv

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
`paragraph-collapsible_panel_item` transformer and friends.

## Testing done
Yup.

## Screenshots
N/A

## Acceptance criteria
- [x] The transformer works as expected

### CODE REVIEW

File: src/site/stages/build/process-cms-exports/tests/transformed-entities/paragraph-collapsible_panel.json [hunk 1]
```diff
@@ -8,165 +8,40 @@
...
-        "baseType": "paragraph",
-        "contentModelType": "paragraph-collapsible_panel_item"
+        "contentModelType": "paragraph-collapsible_panel_item",
+        "entity": {
+          "entityType": "paragraph",
+          "entityBundle": "collapsible_panel_item",
+          "fieldTitle": "Mental health resources for Veterans, service members, and families",
+          "fieldVaParagraphs": [],
+          "fieldWysiwyg": {
+            "processed": ""
```
Comment (bkjohnson) [inline src/site/stages/build/process-cms-exports/tests/transformed-entities/paragraph-collapsible_panel.json]: This `processed` being empty despite the entity having a value like `<p><strong><a href=\"https://www.militaryonesource.mil/mental-health\">Military OneSourc ...` is suspicious

File: src/site/stages/build/process-cms-exports/transformers/paragraph-collapsible_panel_item.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const { getDrupalValue, getWysiwygString } = require('./helpers');
+
+const transform = entity => ({
+  entity: {
+    entityType: 'paragraph',
+    entityBundle: 'collapsible_panel_item',
+    fieldTitle: getDrupalValue(entity.fieldTitle),
+    fieldVaParagraphs: entity.fieldVaParagraphs,
+    fieldWysiwyg: { processed: getWysiwygString(entity.fieldWysiwyg.value) },
```
Comment (bkjohnson) [inline src/site/stages/build/process-cms-exports/transformers/paragraph-collapsible_panel_item.js]: I think we need to be getting `fieldWysiwyg[0].value`.

File: src/site/stages/build/process-cms-exports/tests/transformed-entities/paragraph-collapsible_panel.json [hunk 3]
```diff
@@ -8,165 +8,40 @@
...
-        "baseType": "paragraph",
-        "contentModelType": "paragraph-collapsible_panel_item"
+        "contentModelType": "paragraph-collapsible_panel_item",
+        "entity": {
+          "entityType": "paragraph",
+          "entityBundle": "collapsible_panel_item",
+          "fieldTitle": "Community mental health resources",
+          "fieldVaParagraphs": [],
+          "fieldWysiwyg": {
+            "processed": ""
```
Comment (bkjohnson) [inline src/site/stages/build/process-cms-exports/tests/transformed-entities/paragraph-collapsible_panel.json]: Same concern

**Review by bkjohnson — COMMENTED** [review]
There might be an issue with the transformer

**Comment by cvalarida** [comment]
:man_facepalming: Good catch. Updated!

**Review by bkjohnson — APPROVED** [review]
Looks good! Don't fail us now, Jenkins :crossed_fingers:

**Pull request merged**